benchmarks/hash_function/hash_function_bench.cu

```diff
@@ -34,7 +34,6 @@ template <int32_t Words>
 struct large_key {
   constexpr __host__ __device__ large_key(int32_t seed) noexcept
   {
-#pragma unroll Words
```

Comment:
Could you elaborate why we should delete this line?

Reply:
It's due to a warning we didn't capture previously
```bash
/home/yunsongw/Work/cuCollections/benchmarks/hash_function/hash_function_bench.cu:38: warning: ignoring ‘#pragma unroll Words’ [-Wunknown-pragmas]
   38 |     for (int32_t i = 0; i < Words; ++i) {
      | 
```
Let me update it to `#pragma unroll` which will still do the unrolling without warnings.